frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -59,7 +59,7 @@ export class Wizard {
     await click(view.nextButton);
     if (!ignoreWarnings) {
       try {
-        await browser.wait(until.presenceOf(view.footerError), 1 * SEC);
+        await browser.wait(until.presenceOf(view.footerError), 2 * SEC);
```

Comment:
2 seconds seems a lot. The error should be displayed pretty much immediately, 1 second should already be safe enough, have you observed false negatives?

Reply:
Fixed